jsonrpc/dispatcher.go

```diff
@@ -92,12 +98,19 @@ func (d *Dispatcher) registerEndpoints(store JSONRPCStore) {
 		d.params.chainID,
 		d.params.chainName,
 	}
-	d.endpoints.TxPool = &TxPool{store}
+	d.endpoints.TxPool = &TxPool{
+		store,
+	}
+	d.endpoints.Debug = &Debug{
+		helper,
+		store,
```

Comment:
Just curious about this:

You initialize the `helper` with the `store`, but you instantiate the `Debug` endpoint with both. Could this be avoided?

Reply:
This code has been removed at latest commit